proxies/live/apiproxy/policies/AssignMessage.SelectIdTokenFieldsTokenExchange.xml

```diff
@@ -1,5 +1,6 @@
 <AssignMessage name="AssignMessage.SelectIdTokenFieldsTokenExchange">
-  <IgnoreUnresolvedVariables>false</IgnoreUnresolvedVariables>
```

Comment:
Ignore missing fields here as handled in next policy selectIdTokenIssuer which raises fault. Same behaviour as master

Reply:
Q - if these fields are empty on authcode flow- should this raise error?

Reply:
Currently authcode flow doesn't check issuer as there is currently no conditional nhs login, cis2 logic. So doesn't use these fields.
checks issuer later in JavaScript.SetLoggingFields and uses some info from DecodeJWT.FromExternalIdToken to set provider as cis2 or nhs login, for logging purposes